Interfaces/API/Dirac.py

```diff
@@ -430,7 +430,7 @@ def __forceLocal(self, job):
     else:
       jdl = job
 
-    if not re.search('\[', jdl):
+    if not re.search(r'\[', jdl):
```

Comment:
wouldn't `if '[' not in jdl:` be just as good, but not using the whole regexp machinery.